apps/vs-code-designer/src/package.json

```diff
@@ -67,6 +67,15 @@
           "dark": "assets/dark/CreateNewProject.svg"
         }
       },
+      {
+        "command": "azureLogicAppsStandard.createNewCodeProject",
+        "title": "Create New Code Project...",
```

Comment:
"Create Logic App workspace"?

Reply:
Why change "project" to "workspace"? Eventually, you deploy the project, not a workspace? Also, why do we have `azureLogicAppsStandard.createNewProject` and `azureLogicAppsStandard.createNewCodeProject`? What's the difference?

Let's minimize the sentence casing "Logic App" because we don't want people to think that "logic app" is capitalized. Either "logic app" (the resource) or "Azure Logic Apps" (the service).